src/Root.cpp

```diff
@@ -720,6 +756,27 @@ void cRoot::BroadcastPlayerListsAddPlayer(const cPlayer & a_Player, const cClien
 }
 
 
+
+
+
+size_t cRoot::GetMaxRAMChunks()
```

Comment:
Why? I dislike inline functions and we have so many of them and I think this contributes to slow compilation.